uploadNeuroDB/NeuroDB/Database.pm

```diff
@@ -72,17 +72,16 @@ use constant DEFAULT_DB_PORT => 3306;
 =head3 C<< new(userName => $u, databaseName => $d, hostname => $h, password => $pwd, port =>$port) >>  (constructor)
 
 Create a new instance of this class, without actually trying to connect
-to the database specified. All parameters are required except 'port', which
+to the database specified. All parameters are required except C<port>, which
 defaults to 3306 if not specified. If the user name, database name or host
 name are the empty string, the constructor will call C<die>.
 
-INPUTS: a set of properties for the current database:
-
-- name of the user for the (upcoming) connection
-- name of the database
-- name of the host on which the database resides
-- password for the (upcoming) connection
-- port used for the (upcoming) connection (defaults to 3306 if not provided)
+INPUTS:
+- name of the user for the (upcoming) connection.
```

Comment:
Maybe add two spaces before the -, it might solve the issue with the MD file later on?